panel/src/components/Forms/FormButtons.vue

```diff
@@ -22,29 +58,174 @@
 </template>
 
 <script>
+import dayjs from "dayjs";
+import relativeTime from 'dayjs/plugin/relativeTime';
+dayjs.extend(relativeTime);
+
 export default {
+  data() {
+    return {
+      now: dayjs(),
+      heartbeat: null
+    }
+  },
   computed: {
+    canUnlock() {
+      return this.lock.time + (60 * 2) < this.now.unix();
```

Comment:
Shouldn't this check be handled by the backend? Otherwise it's possible to unlock the page with a manual API request. Instead, the backend could pass this boolean `canUnlock` value back with the lock request.

Another advantage of handling this in the backend: The interval could be made configurable at a later point if needed.

Reply:
👍